src/components/Text.js

```diff
@@ -56,59 +56,65 @@ export const RegularText = styled(Text)`
       line-height: ${normalize(26)};
     `};
 
+  ${(props) =>
+    props.lineThrough &&
+    css`
+      text-decoration: line-through;
+    `};
+
+  ${(props) =>
+    props.underline &&
+    css`
+      text-decoration: underline;
+    `};
```

Comment:
any reason for moving them up here?

Reply:
they need to be defined before the change of their color (which was added in this pr as well)

Reply:
good to know!